Description: UniqueConstraint named and escaped twice
 This patch fixes get_constraint_name in the ANSIConstraintCommon class. It's
 part of the fixes needed for SQLA 0.9.x compat.
Change-Id: I1f1648af48f459bd18f99bb42fa9a272186fb37d
From: Thomas Goirand <thomas@goirand.fr>
Date: Sun, 2 Mar 2014 13:07:39 +0800

diff --git a/migrate/changeset/ansisql.py b/migrate/changeset/ansisql.py
index 495b197..c5ff405 100644
--- a/migrate/changeset/ansisql.py
+++ b/migrate/changeset/ansisql.py
@@ -275,7 +275,7 @@ class ANSIConstraintCommon(AlterTableVisitor):
             ret = cons.name
         else:
             ret = cons.name = cons.autoname()
-        return self.preparer.quote(ret, cons.quote)
+        return ret
 
     def visit_migrate_primary_key_constraint(self, *p, **k):
         self._visit_constraint(*p, **k)
-- 
1.7.10.4

